### DISCUSSION TITLE
String attribute not being populated on specific model

### DISCUSSION THREAD

**Discussion by JamesHutchison** [fastapi/sqlmodel#736] [Questions]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the SQLModel documentation, with the integrated search.
- [X] I already searched in Google "How to X in SQLModel" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to SQLModel but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to SQLModel but to [SQLAlchemy](https://github.com/sqlalchemy/sqlalchemy).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

I don't have a reproducing example yet, but for some reason this model is ignoring the string attributes redeemer_name and redeemer_org and setting them to `None`

```python


class SQLAccessRedemption(HeavyModel, table=True):
    __tablename__ = "access_redemptions"

    id: RedemptionId = Field(default_factory=lambda: uuid4().hex, primary_key=True)
    invite_id: InviteId = HeavyForeignKeyField(foreign_key="access_invites.id")
    invite: "SQLAccessInvite" = Relationship(back_populates="redemptions")
    redeemer_name: str
    redeemer_org: str
    redeemed_by_id: UserId = HeavyForeignKeyField(foreign_key="users.id")
    redeemed_by: "SQLUser" = HeavyRelationship(back_populates="redemptions")
    redeemed_at: datetime.datetime = HeavyDateTime()
    created_key_id: EncryptedKeyId = HeavyForeignKeyField(foreign_key="encrypted_keys.key_id")
    created_key: SQLEncryptedKey = HeavyRelationship()
    expires: datetime.datetime | None = HeavyDateTime()


...

        access_redemption = SQLAccessRedemption(
            invite_id=access_invite.id,
            redeemed_by_id=self.user_id,
            redeeemer_name=redeemed_by_name,  # not working
            redeeemer_org=redeemed_by_org,  # not working
            created_key_id=created_key.sql_encrypted_key.key_id,
            expires=self._ensure_minimum_time(access_invite.expires, access_invite.minimum_time),
        )
        # workaround is to add this after creating the object
        access_redemption.redeemer_name = redeemed_by_name
        access_redemption.redeemer_org = redeemed_by_org


```

The custom classes / functions don't affect anything.

Creating a minimal example with just strings in a table, it passes, so there's something specific about this class that's breaking it. The other fields are set just fine. `redeemed_by_org` and `redeemed_by_name` are just strings.

### Description

Instantiating a specific class

### Operating System

Linux

### Operating System Details

Debian

### SQLModel Version

0.0.12 and 0.0.14

### Python Version

3.11.4

### Additional Context

I'll see if I can boil this down to a specific, reproducible example later, but it will be time consuming. If someone already knows the cause or about this then that would be helpful.

**Comment by YuriiMotov** [comment]
`redeeemer_name` (3 `e` instead of 2)
`redeeemer_org` (3 `e` instead of 2)
